README.md

```diff
@@ -103,7 +103,7 @@ Widget build(BuildContext context) {
 
 To use Azure Maps you will need to [setup an account and get a subscription key](https://docs.microsoft.com/en-us/azure/azure-maps/quick-demo-map-app)
 
-### Open Street Map provider
+### Open Street Maps Provider
```

Comment:
I think these titles were meant to be sentence-case. https://developers.google.com/style/headings